tests/resources/test_templates.py

```diff
@@ -37,6 +37,3 @@ def test_template_assignment():
     template = ProcessTemplate("Dry", conditions=[[humidity_template, RealBounds(0.5, 0.65, "")]])
     ProcessSpec("Dry a polymer", template=template, conditions=[
         Condition("Humidity", value=NominalReal(0.6, ""), template=humidity_template)])
-    with pytest.raises(RuntimeWarning):
-        ProcessSpec("Dry a polymer", template=template, conditions=[
-                   Condition("Humidity", value=NominalReal(0.7, ""))])
```

Comment:
I believe this is because we removed much of the client-side validation from Taurus this morning, so this no longer raises a warning.